Sources/Build/BuildPlan.swift

```diff
@@ -1978,3 +2006,14 @@ fileprivate extension Triple {
         isLinux() || arch == .wasm32
     }
 }
+
+// FIXME: move this to TSC
+extension Triple: Hashable {
+    public func hash(into hasher: inout Hasher) {
+        hasher.combine(self.tripleString)
```

Comment:
This probably includes the others, but it doesn't hurt to include them all.

Reply:
right, wanted to be extra careful